src/main/groovy/com/avioconsulting/mule/linter/model/pom/PomFile.groovy

```diff
@@ -59,8 +61,31 @@ class PomFile extends ProjectFile {
         return parser.getNodeLineNumber(pomProperties)
     }
 
+    PomPlugin getPlugin(String groupId, String artifactId) {
+        PomPlugin plugin
+        GPathResult pluginPath = pomXml.build.plugins.plugin.find {
+            it.groupId == groupId && it.artifactId == artifactId
+        } as GPathResult
+
+        if (pluginPath != null && pluginPath.size() > 0) {
+            plugin = new PomPlugin(pluginPath, this)
+        }
+        return plugin
+    }
+
+    MunitMavenPlugin getMunitPlugin() {
+        PomPlugin pp = getPlugin(MunitMavenPlugin.GROUP_ID, MunitMavenPlugin.ARTIFACT_ID)
+        return pp == null ? null : new MunitMavenPlugin(pp.pluginXml, this)
+    }
+
     private GPathResult getPomProperties() {
```

Comment:
This isn't used and we discussed not exposing the full properties GPathResult

Reply:
the getPomProperties?  that method is private, and only used by getPomProperty... 

I hadn't changed the getPomProperties...

Reply:
Used by 2 methods (getPomProperty and getPropertyLineNo)